CODEOWNERS

```diff
@@ -127,6 +127,8 @@
 /Workbooks/Azure*Private*MEC/** @microsoft/azure-private-mec-workbooks
 /Workbooks/Azure*API*Management/** @microsoft/azure-api-management
 
+/Workbooks/Azure*Logic*Apps/** @Azure/azure-logicapps-team
```

Comment:
there you go, codeowners reports this as invalid:

> Unknown owner on line 130: make sure the team @Azure/azure-logicapps-team exists, is publicly visible, and has ?> write access to the repository
>  …Workbooks/Azure*Logic*Apps/** @Azure/azure-logicapps-team

so if you don't already have a team in @microsoft org you need to add one, and then let me know what it is so i can add that team to have write access to the repo.

technically this shoudl have been one of the first steps as it is in the contributing docs:
https://github.com/microsoft/Application-Insights-Workbooks/blob/master/CONTRIBUTING.md#recommended-setup-for-contributing

> If you are a Microsoft employee:

>Ensure your github user is already a member of the [Microsoft organization](https://repos.opensource.microsoft.com/orgs)
> Look at the CODEOWNERS file to find if there is already a team set up for the area you are looking to contribute.
> If there is an existing team, ask the owners of that team to add you there if appropriate.
> if there is not an existing team, and you're looking to add workbook templates to new areas, in most cases you'll need to create a new team in the Microsoft organization in GitHub.
> If you created a new team, or you don't fall under one of the existing teams, send mail to azmonworkbooks with all the relavant information, including GitHub team name and details about the area you want to contribute to, so we can direct your request to the appropriate teams.

Reply:
Thanks! I did read those docs, but thought perhaps [these other MSFT orgs](https://github.com/enterprises/microsoftopensource/organizations) might be part of the 'in most cases' 🙂 Will send the new team reference soon